### PR TITLE
ES samples improvements

### PR DESCRIPTION
**Pull request by sebgl**
Some minor improvements in ES samples.

### CODE REVIEW

**Comment by sebgl** [comment]
Jenkins test this please.

File: operators/config/samples/elasticsearch/elasticsearch.yaml [hunk 1]
```diff
@@ -4,11 +4,15 @@ kind: Elasticsearch
 metadata:
   name: elasticsearch-sample
 spec:
-  version: "7.1.0"
+  version: "7.2.0"
```
Comment (anyasabo) [inline operators/config/samples/elasticsearch/elasticsearch.yaml]: Should we just go all the way to 7.2.1?

Comment (sebgl) [reply operators/config/samples/elasticsearch/elasticsearch.yaml]: 👍

Comment (sebgl) [reply operators/config/samples/elasticsearch/elasticsearch.yaml]: Let's do it for all docs actually, including quickstart.
I'll open another PR.

**Pull request merged**